src/picknik_ur_gazebo_config/launch/sim/hardware_sim.launch.py

```diff
@@ -59,7 +73,15 @@ def path_pattern_change_for_ignition(urdf_string):
 
 
 def generate_simulation_description(context, *args, **settings):
-    world_name = settings.get("gazebo_world_name", "space_station.sdf")
+    nodes = []
+    is_test = LaunchConfiguration("is_test").perform(context).lower() == "true"
```

Comment:
If we're not running this configuration through our CI pipeline, we can get rid of the `is_test` argument as well as the toggle on world name key below.